sycl/plugins/unified_runtime/ur/adapters/level_zero/ur_level_zero_common.hpp

```diff
@@ -23,5 +28,158 @@
 #include "ur_level_zero_queue.hpp"
 #include "ur_level_zero_sampler.hpp"
 
+template <class To, class From> To ur_cast(From Value) {
+  // TODO: see if more sanity checks are possible.
+  assert(sizeof(From) == sizeof(To));
+  return (To)(Value);
+}
+
+// Trace an internal PI call; returns in case of an error.
+#define UR_CALL(Call)                                                          \
```

Comment:
similarly, shouldn't this just replace PI_CALL?

Reply:
@smaslov-intel : same, this will go away. It is just to not change all at once. However, of course, we could do those changes as part of these PR. I dont have a preference. what do you think?